docs/installation/expansion.rst

```diff
@@ -65,12 +49,23 @@ The MetalK8s GUI provides a simple form for that purpose.
    - **SSH Key Path**: the path to the private key generated in
      :ref:`this procedure <Bootstrap SSH Provisioning>`
    - **Sudo required**: whether the SSH deployment will need ``sudo`` access
-   - **Roles/Workload Plane**: check this box if the new Node should receive
-     workload applications
-   - **Roles/Control Plane**: check this box if the new Node should run
-     control-plane services
-   - **Roles/Infra**: check this box if the new Node should run
-     infra services
+   - **Roles/Workload Plane**: if you select this box, you will remove all the
```

Comment:
Personally, I would change with the following:

Roles/Workload Plane: any workload application can run on this node.
Roles/Control Plane: Master and etcd services can run on this node.
Roles/Infra: Infra services can run on this node.

Reply:
```
Roles/Workload Plane: enable any workload application
Roles/Control Plane: enable master and etcd services
Roles/Infra: enable infra services
```